### PR TITLE
fix: Update Security guide

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Pull Request

## Description

Specify the installation of an additional extension for the correct display of the documentation

## Type of change

- [x] Documentation update

**Test Configuration**:
* Java: v17
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] My code meets the required code coverage for lines (90% and above)
- [ ] My code meets the required code coverage for branches (80% and above)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 This document outlines the procedures for addressing vulnerabilities, the supported versions of LPVS, security requirements, and the recommended practices for developing secure code.
 
+!!! note
```
Comment (o-kopysov) [inline .github/SECURITY.md:17]: Use standard notes used in GitHub.

**Review by o-kopysov — CHANGES_REQUESTED** [review]
Please check my comment.

**Comment by t-naumenko** [comment]
Let's try to use marmaid which is already integrated in GitHub by default:
```
## 3. Security requirements
```mermaid
graph LR
    A[Security requirements] --> B[Confidentiality]
    A --> C[Integrity]
    A --> D[Availability]
    A --> E[Access control]

    E --> F[Identification]
    E --> G[Authentication]
    E --> H[Authorization]

    B --> I[User privacy maintained]
    B --> J[Non-public data is kept confidential]
    B --> K[All data is confidential]
    B --> L[HTTPS: data in motion]

    C --> L
    C --> M[Authorization via GITHUB]
    C --> N[Data modification requires authorization]

    D --> O[Multiple backups]
    D --> P[Restore after DDoS]
```
---
```

## 4. Security Software life cycle processes

```mermaid
graph LR
    A[Security Software Life Cycle Processes] --> B[Certification & Controls]
    A --> C[Security in Maintenance]
    A --> D[Key Developers Know How to Develop Secure Software]
    A --> E[Security in Integration & Verification]
    A --> F[Infrastructure Management]
    A --> G[Security in Design]

    B --> H[CII Best Practices Badge]
    B --> I[OpenSSF Score Card]
    C --> J[Auto-detect Public Vulnerabilities]
    C --> K[Rapid Update]
    F --> L[Development & Test Environments Protected from Attack]
    F --> M[CI Automated Test Environment Does Not Have Protected Data]
    E --> N[Style Checking Tools]
    E --> O[Source Code Weakness Analyzer]
    E --> P[Free/Libre/Open Source Software]
    E --> Q[Negative Testing]
    E --> R[Unit Test Coverage >75%]
    G --> S[Simple Design]
    G --> T[Memory-Safe Languages]
    G --> U[Secure Design Includes S&S]
```

---
```

**Review by o-kopysov — APPROVED** [review]
LGTM

**Review by t-naumenko — APPROVED** [review]
LGTM

**Pull request merged**